modules/menu/views/license_info.php

```diff
@@ -0,0 +1,32 @@
+<?php
+if (($system_version = @file_get_contents('/etc/op5-monitor-release')) !== FALSE) { ?>
+        <p>Current OP5 Monitor System version: <strong><?php echo str_replace('VERSION=', '', $system_version); ?></strong></p>
+<?php } ?>
+
+<h1>Service &amp; Support</h1>
+<p>Detailed information regarding Service and Support is covered in the general
+mandatory OP5 Service and Support agreement.This needs to be in place prior to
```

Comment:
Can you ask product owner to verify the text on the rendered page? Maybe some wording should be changed to "ITRS" etc.

Reply:
Will do. Thanks!